api/views/session_views.py

```diff
@@ -224,6 +228,39 @@ def get(self, request, quiz_id):
         return JsonResponse({"quiz_sessions": quiz_sessions_data})
 
 
+@extend_schema(tags=["Session Management"])
+class QuizSessionLogView(APIView):
+    permission_classes = []
+
+    @extend_schema(
+        request=AddQuizSessionLogSerializer(),
+        responses={
+            201: OpenApiResponse(description="Log created and added successfully"),
+            400: OpenApiResponse(description="Bad Request"),
+            404: OpenApiResponse(description="Object not found"),
+        },
+    )
+    def post(self, request):
+        serializer = AddQuizSessionLogSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(
+                {"message": "Log entry successfully created."}, status=status.HTTP_201_CREATED
```

Comment:
Instead of returning a message, we should return the attributes of the newly created row; that's often the common practice, and you can update your tests to verify that everything has been saved successfully.